### PR TITLE
Update scalariform

### PR DESCRIPTION
**Pull request by raboof**
Refs #23567,#23778

### CODE REVIEW

File: project/plugins.sbt [hunk 1]
```diff
@@ -9,7 +9,7 @@ resolvers += "Bintray Jcenter" at "https://jcenter.bintray.com/"
 addSbtPlugin("com.typesafe.sbt" % "sbt-multi-jvm" % "0.3.8")
 //#sbt-multi-jvm
 
-addSbtPlugin("org.scalariform" % "sbt-scalariform" % "1.6.0")
+addSbtPlugin("org.scalariform" % "sbt-scalariform" % "1.8.0")
```
Comment (ktoso) [inline project/plugins.sbt]: That's the version we'll be able to use on sbt 1 too right?

**Review by ktoso — APPROVED** [review]
LGTM, let's do it ;)

**Review by johanandren — APPROVED** [review]

File: project/plugins.sbt (hunk 1 above)
Comment (raboof) [reply project/plugins.sbt]: Yes, that was indeed the trigger to finally push this (though I also like the new `NewlineAtEndOfFile` setting :) )

**Review by patriknw — APPROVED** [review]
LGTM

**Comment by raboof** [comment]
Sorry about the merge conflicts!

**Pull request merged**